### DISCUSSION TITLE
Thoughts on passivity-based control module?

### DISCUSSION THREAD

**Discussion by Mark-Yeatman** [python-control/python-control#732] [Ideas]
Hello!

I've used this package on and off for a while, and I was wondering if anyone had done any work on anything explicitly related to passivity-based control? It seems like Matlab has a tool, https://www.mathworks.com/help/control/passivity-and-sector-bounds.html?s_tid=CRUX_lftnav, I'd be interested in trying to replicate some of this functionality.

**Comment by murrayrm** [comment]
I'm not aware of anything that has been done on passivity in python-control.  Definitely feel free to generate a PR (in draft form if you would like some feedback on the way).

**Reply by Mark-Yeatman** [reply]
Awesome, will do.
